treeherder/config/settings.py

```diff
@@ -393,70 +472,36 @@
 BUILDAPI_RUNNING_CHUNK_SIZE = 500
 BUILDAPI_BUILDS4H_CHUNK_SIZE = 500
 
+BUILDAPI_PENDING_URL = "https://secure.pub.build.mozilla.org/builddata/buildjson/builds-pending.js"
+BUILDAPI_RUNNING_URL = "https://secure.pub.build.mozilla.org/builddata/buildjson/builds-running.js"
+BUILDAPI_BUILDS4H_URL = "https://secure.pub.build.mozilla.org/builddata/buildjson/builds-4hr.js.gz"
+ALLTHETHINGS_URL = "https://secure.pub.build.mozilla.org/builddata/reports/allthethings.json"
```

Comment:
Happy for most of these to be moved to where they are used, too (can be in another PR if preferred)